File: ProjectAPI/ProjectAPI/Model/Request.cs
using Microsoft.Build.Evaluation;
using Microsoft.VisualBasic;
using System.Data;
using System.Globalization;
using System.Reflection.Metadata.Ecma335;

namespace ProjectAPI.Model
{
    public class Request
    {
        public int RequestId { get; set; }
        //public string UserName { get; set; }

         public User?  User { get; set; }

        public int UserId { get; set; }

        public Project? Project { get; set; }

        public int ProjectId { get; set; }

      //  public string ReasonForTraveling { get; set; }


        public string TypeOfBooking { get; set; }

        public virtual User? Manager { get; set; }
        public int ManagerId { get; set; }

        public string CreatedBy { get; set; }

        public DateTime CreatedDate { get; set; } = DateTime.Now;

        public string? ModifiedBy { get; set; }

        public DateTime? ModifiedDate { get; set; }

        public Boolean IsActive { get; set; } = true;



    }
}

File: src/Request.ts
export class Request{

    firstName:string;
    userId:number;
    projectId:number;
    typeOfBooking:string;
managerId:number;
    middleName:string;
    lastName:string;
    project:string;
    department:string;
    reasonForTravel:string;
    travelMode:string;
    createdBy:string;
    createdDate:Date;
    modifiedBy:string;
    modifiedDate:Date;
    isActive:boolean;

    constructor()
    {
        this.firstName="",
        this.userId=0;
        this.projectId=0;
        this.typeOfBooking="",
        this.managerId=0;
        this.middleName="",
    this.lastName="",
    this.project="",
    this.department="",
    this.reasonForTravel="",
    this.travelMode="",
    this.createdBy="",
    this.createdDate=new Date,
    this.modifiedBy="",
    this.modifiedDate=new Date,
    this.isActive=false;
    }


}